packages/flex-plugin-scripts/src/config/devServer/pluginServer.ts

```diff
@@ -73,40 +83,32 @@ export const _getRemotePlugins = (token: string, version: string): Promise<Plugi
  * @param remotePlugins   the plugins returned from Flex
  * @private
  */
-export const _rebasePlugins = (remotePlugins: Plugin[]) => {
-  return _getLocalPlugins()
+export const _mergePlugins = (localPlugins: Plugin[], remotePlugins: Plugin[]) => {
```

Comment:
```
export const _mergePlugins = (localPlugins: Plugin[], remotePlugins: Plugin[]) => {
  const deduped = remotePlugins.filter(r => !localPlugins.some(l =>l.name === r.name));
  return [... localPlugins, ...deduped];
});

Reply:
I changed it to `const deduped = remotePlugins.filter(p => p.phase >=3);` here because i have the remote filtering in default and the phase filtering in here